valhalla/baldr/graphreader.h

```diff
@@ -819,6 +866,9 @@ class GraphReader {
   std::unordered_set<GraphId> _404s;
 
   std::unique_ptr<TileCache> cache_;
+
+  // TODO: delete me when incident singleton is ready
+  bool simulate_incidents_;
```

Comment:
in place of real incidents for now just to enable or disable the logic for fetching incidents